services/database.js

```diff
@@ -77,10 +77,22 @@ function Database() {
       };
     }
 
+    let { dbConnectionUrl, dbHostname } = options;
+    if (databaseDialect === 'mssql' && dbConnectionUrl && dbConnectionUrl.includes('\\')) {
+      const [prefix, suffix] = dbConnectionUrl.split('\\');
+      const indexOfColon = suffix.indexOf(':');
+      dbConnectionUrl = `${prefix}${suffix.substring(indexOfColon)}`;
+      connectionOptionsSequelize.dialectOptions.instanceName = suffix.substring(0, indexOfColon);
+    } else if (databaseDialect === 'mssql' && options.dbHostname && options.dbHostname.includes('\\')) {
+      const [hostname, instanceName] = dbConnectionUrl.split('\\');
+      dbHostname = hostname;
+      connectionOptionsSequelize.dialectOptions.instanceName = instanceName;
+    }
```

Comment:
It happens with Sql Server Express if I'm not mistaken. On SQL Server you can have an instance name which is next to the hostname with a `\`

Reply:
This code should be tested, especially if we cannot really test it in a Windows environment.
I think it should be easy to test this with a few `dbConnectionUrl` formats (and `dbHostname` for the second case)

It would also make this logic easy to understand.